src/getting-started/assets/index.html

```diff
@@ -323,7 +323,7 @@ <h3 class="font-weight-light">Why do I see the JDK errors?</h3>
                     </p>
                     <h3 class="font-weight-light">Can I run my Java 8 project with JDK 1.8?</h3>
                     <p>
-                      Yes. JDK 11 requirement is just for running VS Code Java extension itself, you can still configure a different runtime <a href="command:java.helper.openUrl?%22https%3A%2F%2Fcode.visualstudio.com%2Fdocs%2Fjava%2Fjava-project%23_jdk-for-projects%22">JDK for your project</a> via user setting <a href="command:workbench.action.openSettings?%22java.configuration.runtimes%22">"java.configuration.runtimes"</a>. The extension will pick a mapping JDK to compile/run your project according to the compiler version specified by the project build file.
+                      Yes. JDK 11 requirement is just for running <b>Language Support for Java by Red Hat</b> extension itself, you can still configure a different runtime <a href="command:java.helper.openUrl?%22https%3A%2F%2Fcode.visualstudio.com%2Fdocs%2Fjava%2Fjava-project%23_jdk-for-projects%22">JDK for your project</a> via user setting <a href="command:workbench.action.openSettings?%22java.configuration.runtimes%22">"java.configuration.runtimes"</a>. The extension will pick a mapping JDK to compile/run your project according to the compiler version specified by the project build file.
```

Comment:
There seems to be a missing "the" before the extension name (though this issue was already present in the previous text).

Reply:
good finding